### DISCUSSION TITLE
i3 on Windows!

### DISCUSSION THREAD

**Discussion by dillacorn** [dillacorn/win-glaze-dots#2] [Announcements]
Made enough revisions that I feel a new release is deserved.

I've created a discussion topic on why Windows remains necessary for most multiplayer focused gamers today.

https://github.com/dillacorn/win-glaze-dots/discussions/1

<hr /><em>This discussion was created from the release <a href='https://github.com/dillacorn/win-glaze-dots/releases/tag/v1.1'>i3 on Windows!</a>.</em>

**Comment by aryajoy** [comment]
Windows is a user-friendly operating system developed by Microsoft, known [penumbra ffxiv](https://xivlauncher.com/penumbra-ffxiv/) for its wide compatibility.
It powers millions of PCs worldwide, offering tools for work, gaming, and creativity.

**Reply by dillacorn** [reply]
I haven't heard of "Penumbra" modding tool. Thanks for the suggestion I'll look into it! :) ~ (I don't play final fantasy that's probably why)
